source/flow.hpp

```diff
@@ -13,11 +13,16 @@ class ISender;
 class IFlow : public Identifiable {
 public:
     virtual void start(Time time) = 0;
-    virtual Time try_to_generate() = 0;
+    // Represents process of appearing data on Sender
+    // Uses event Generate
+    virtual Time create_new_data_packet() = 0;
+    // Puts data on corresponding device sending buffer according to the internal state
+    // Can be scheduled by event or called on some condition
+    virtual Time add_data_to_device() = 0;
 
     // Update the internal state according to some congestion control algorithm
-    // Call try_to_generate upon the update
-    virtual void update() = 0;
+    // Puts data available for sending on corresponding device
```

Comment:
Lets make comments more informative
```suggestion
    // Calls when data available for sending on corresponding device
```